mlrun/serving/server.py

```diff
@@ -451,8 +451,14 @@ def push_error(self, event, message, source=None, **kwargs):
                 f"got error from {source} state:\n{event.body}\n{message}"
             )
         if self._server and self._server._error_stream_object:
-            message = format_error(self._server, self, source, event, message, kwargs)
-            self._server._error_stream_object.push(message)
+            try:
+                message = format_error(
+                    self._server, self, source, event, message, kwargs
+                )
+                self._server._error_stream_object.push(message)
+            except BaseException as ex:
```

Comment:
Reading the second comment in [this one](https://stackoverflow.com/questions/44655854/what-is-the-difference-between-except-and-except-baseexception) makes me think we should catch `Exception` and not `BaseException`, WDYT ?